HUGODEHSIGN/my-otaku-opinions
# Style handlebars pages and add cards to carousel

## Purpose

Please provide a detailed explanation of the PR. What existing problem does it solve?

Fixes #98 

## Changes

Please provide an overview of code changes, especially key files and areas of significant change.

- path/to/file
  - public/css/carousel.css
  - public/css/style.css
  - views/example.card.html
  - views/homepage.handlebars
  - views/layouts/main.handlebars
  - views/login.handlebars
  - views/partials/animeCard.handlebars
  - views/signup.handlebars

## Steps to Reproduce or Test

Demonstrate that the pull request is functional. For example: the exact commands to be run and their output; screenshots if the PR modifies the UI.

- clone the branch
- npm i in terminal
- npm start in terminal
- got to localhost:3001 in browser
- view the webpage, make sure to visit every page
- confirm nothing was missed or broken
- I'm so sorry for how long this is....
